### PR TITLE
Adds the ability to specify a separate RSI path in StackVisualizer

### PR DESCRIPTION
**Pull request by SweptWasTaken**
## About the PR 

Title. Just allows reduction of RSI bloat.

### CODE REVIEW

File: Content.Client/GameObjects/Components/StackVisualizer.cs [hunk 1]
```diff
@@ -102,9 +108,14 @@ public override void InitializeEntity(IEntity entity)
                 && _spriteLayers.Count > 0
                 && entity.TryGetComponent<ISpriteComponent>(out var spriteComponent))
             {
+                if (_spritePath == null)
```
Comment (Ygg01) [inline Content.Client/GameObjects/Components/StackVisualizer.cs:111]: I think C# has a better construct  `_spritePath ?? = spriteComponent.BaseRSI!.Path!`

Comment (DrSmugleaf) [reply Content.Client/GameObjects/Components/StackVisualizer.cs:111]: [??= operator](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/null-coalescing-operator)

**Comment by SweptWasTaken** [comment]
Applied reviews

**Comment by PJB3005** [comment]
Wait fuck no ??= is the wrong operator to use because that mutates the visualizer.

**Comment by SweptWasTaken** [comment]
> 
> 
> Wait fuck no ??= is the wrong operator to use because that mutates the visualizer.

wdym mutates it

**Comment by PJB3005** [comment]
Fixed that by 2ec030407202c504cc4b46a3b73c7ac3e2566476

Answered your question on discord.

**Pull request merged**